dora/src/cpu/x64/reg.rs

```diff
@@ -67,6 +67,19 @@ pub const XMM13: FReg = FReg(13);
 pub const XMM14: FReg = FReg(14);
 pub const XMM15: FReg = FReg(15);
 
+pub const VEXM_0F: u8 = 1;
+pub const VEXM_0F38: u8 = 2;
+pub const VEXM_0F3A: u8 = 3;
+
+pub const VEXL_Z: u8 = 0;
+pub const VEXL_128: u8 = 0;
+pub const VEXL_256: u8 = 1;
+
+pub const VEXP_NONE: u8 = 0;
+pub const VEXP_66: u8 = 1;
+pub const VEXP_F3: u8 = 2;
+pub const VEXP_F2: u8 = 3;
```

Comment:
Is there a more appropriate place for these constants?

Reply:
They could just go into `masm/x64.rs` or `x64/asm.rs` since all the other code doesn't really care which shift operations are used.

Reply:
I would move this to `masm/x64.rs` or `x64/asm.rs`.

Reply:
Will do.